podcasts/MultiSelectHelper.swift

```diff
@@ -318,6 +321,23 @@ class MultiSelectHelper {
         actionDelegate.multiSelectActionCompleted()
     }
 
+    private class func share(actionDelegate: MultiSelectActionDelegate, view: UIView?) {
+        guard let episode = actionDelegate.multiSelectedBaseEpisodes().first as? Episode else {
```

Comment:
I wonder if we should add an extension to the BaseEpisode that adds a "shareable" property that we can use instead of hardcoding as? Episode. 

Something like: 
```
extension BaseEpisode {
    var shareable: Bool {
        self is Episode
    }
}
```